### ISSUE TITLE
Missing values in state when creating a managed Kubernetes resource in Hong Kong or Singapore

### ISSUE THREAD

**Issue by danbadge** [terraform-providers/terraform-provider-alicloud#1281]
### Terraform Version
v0.11.13

### Affected Resource(s)
- alicloud_cs_managed_kubernetes

### Terraform Configuration Files
```hcl
provider "alicloud" {
  region  = "cn-hongkong"
  version = "~> 1.41"
}

resource "alicloud_vpc" "vpc" {
  name       = "example-vpc"
  cidr_block = "10.10.0.0/21"
}

resource "alicloud_vswitch" "vsw" {
  name              = "example-vswitch"
  vpc_id            = "${alicloud_vpc.vpc.id}"
  cidr_block        = "10.10.0.0/24"
  availability_zone = "cn-hongkong-b"
}

resource "alicloud_cs_managed_kubernetes" "kubernetes" {
  name                  = "example-kubernetes"
  new_nat_gateway       = true
  vswitch_ids           = ["${alicloud_vswitch.vsw.id}"]
  worker_instance_types = ["ecs.n1.medium"]
  worker_numbers        = [2]
  password              = "mypassword"
  pod_cidr              = "172.16.0.0/16"
  service_cidr          = "172.19.0.0/20"
  install_cloud_monitor = true
  slb_internet_enabled  = true
}
```

### Expected Behavior
A managed Kubernetes service is created successfully and when I re-run `terraform plan` or `terraform apply` it doesn't want to recreate the cluster every time.

### Actual Behavior
A managed Kubernetes service is created successfully but when I re-run `terraform plan` or `terraform apply` terraform wants to consistently recreate the whole cluster because it thinks there are changes.

### Steps to Reproduce
1. Run `terraform apply` for the first time
2. After that's finished run `terraform plan` and it will consistently produce the following output (without the redacted IDs):
```
alicloud_cs_managed_kubernetes.kubernetes (new resource required)
      id:                          "c38c849..." => <computed> (forces new resource)
      availability_zone:           "cn-hongkong-b" => <computed>
      install_cloud_monitor:       "true" => "true"
      name:                        "example-kubernetes" => "example-kubernetes"
      name_prefix:                 "Terraform-Creation" => "Terraform-Creation"
      new_nat_gateway:             "false" => "false"
      password:                    <sensitive> => <sensitive> (attribute changed)
      pod_cidr:                    "172.16.0.0/16" => "172.16.0.0/16"
      security_group_id:           "sg-j6c0rm...." => <computed>
      service_cidr:                "172.19.0.0/20" => "172.19.0.0/20"
      slb_internet_enabled:        "true" => "true"
      vpc_id:                      "vpc-j6cg..." => <computed>
      vswitch_ids.#:               "1" => "1"
      vswitch_ids.0:               "" => "vsw-j6cttb..." (forces new resource)
      worker_disk_category:        "cloud_efficiency" => "cloud_efficiency"
      worker_disk_size:            "40" => "40"
      worker_instance_charge_type: "PostPaid" => "PostPaid"
      worker_instance_types.#:     "1" => "1"
      worker_instance_types.0:     "" => "ecs.n1.medium" (forces new resource)
      worker_nodes.#:              "2" => <computed>
      worker_numbers.#:            "1" => "1"
      worker_numbers.0:            "2" => "2"
```

### Important Factoids
* This behaviour occurs in Hong Kong and Singapore regions but not in Shanghai
* As far as I can tell the Alicloud JSON API response for a managed Kubernetes cluster in HK and Singapore, when compared with Shanghai, is missing the property: `parameters.VSwitchId` and the instance type key is spelt differently: `parameters.WorkerInstanceTypes` instead of the expected `parameters.WorkerInstanceType`. I believe these two lines are the issue https://github.com/terraform-providers/terraform-provider-alicloud/blob/master/alicloud/resource_alicloud_cs_managed_kubernetes.go#L425

**Comment by xh4n3** [comment]
we spotted this issue, will fix asap.

**Comment by xh4n3** [comment]
for workaround, comment out the vswitch_ids/worker_instance_types temporarily in your terraform template.

**Comment by xh4n3** [comment]
this is fixed in v1.48.0, please have a try.
